src/navigation/MoreStack.tsx

```diff
@@ -13,36 +13,29 @@ import ENLocalDiagnosisKeyScreen from "../More/ENLocalDiagnosisKeyScreen"
 import ExposureListDebugScreen from "../More/ExposureListDebugScreen"
 import LanguageSelection from "../More/LanguageSelection"
 
-import { Screens, Stacks } from "./index"
+import { MoreStackScreens, MoreStackScreen } from "./index"
 
-const Stack = createStackNavigator()
+type MoreStackScreenParams = {
+  [key in MoreStackScreen]: undefined
+}
+const Stack = createStackNavigator<MoreStackScreenParams>()
 
 const SCREEN_OPTIONS = {
   headerShown: false,
 }
 
-type MoreStackRouteName =
-  | "Settings"
-  | "About"
-  | "Licenses"
-  | "ENDebugMenu"
-  | "LanguageSelection"
-  | "AffectedUserFlow"
-  | "ExposureListDebugScreen"
-  | "ENLocalDiagnosisKey"
-
 interface MoreStackRouteState {
```

Comment:
@amandabeiner Some of this code is being removed in https://github.com/Path-Check/gaen-mobile/pull/20 so it may make sense to rebase onto that once it's merged? Not sure what is best here. Lmk what you think!

Reply:
Actually scratch that, I'll rebase onto this!